design-system/package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "@babel/runtime": "^7.19.0",
     "@babel/runtime-corejs3": "^7.19.1",
+    "@emotion/react": "^11.4.0",
```

Comment:
Is this needed? 🤔

Reply:
Ah I saw the [comment](https://github.com/commercetools/ui-kit/pull/2391#issuecomment-1400043487) now